docs/add-ins/get-and-set-recurrence.md

```diff
@@ -173,7 +173,7 @@ var recurrenceType = recurrence.recurrenceType;
 [RecurrenceChanged event](/javascript/api/office/office.eventtype)
 
 [getAsync link]: /javascript/api/outlook_1_7/office.recurrence#getasync-options--callback-
-[item.recurrence link]: /javascript/api/outlook_1_7/office.item#recurrence
+[item.recurrence link]: /javascript/api/outlook_1_7/office.item
```

Comment:
Should point to this instead: /office/dev/add-ins/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item?view=office-js#nullable-recurrence-recurrence

Reply:
@ElizabethSamuel-MSFT I'll make the change you're proposing. But ultimately shouldn't the `recurrence` property be documented in [Office.item](https://docs.microsoft.com/en-us/javascript/api/outlook_1_7/office.item), so that this link can point to that API reference article, and not to the article that you're suggesting in the `-pr` repo?